var/spack/repos/builtin/packages/kim-api/package.py

```diff
@@ -0,0 +1,47 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+#
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class KimApi(CMakePackage):
+    """OpenKIM is an online framework for making molecular simulations
+       reliable, reproducible, and portable. Computer implementations of
+       inter-atomic models are archived in OpenKIM, verified for coding
+       integrity, and tested by computing their predictions for a variety
+       of material properties. Models conforming to the KIM application
+       programming interface (API) work seamlessly with major simulation
+       codes that have adopted the KIM API standard.
+    """
+    homepage = "https://openkim.org/"
+    url      = "https://github.com/openkim/kim-api/archive/v1.9.7.tar.gz"
+    git      = "https://github.com/openkim/kim-api"
+
+    version('develop', branch='master')
+    version('2.0rc1', commit="c2ab409ec0154ebd85d20a0a1a0bd2ba6ea95a9c") 
```

Comment:
It would be good to have the most recent stable release as the default version instead of a release candidate.

Reply:
Sure, but they are switching to CMake for version 2.

Reply:
Ah, that's a good reason to stick with 2 I suppose.